data/moves.js

```diff
@@ -15358,6 +15362,7 @@ exports.BattleMovedex = {
 		desc: "Prevents the target from switching out. The target can still switch out if it is holding Shed Shell or uses Baton Pass, Parting Shot, U-turn, or Volt Switch. If the target leaves the field using Baton Pass, the replacement will remain trapped. The effect ends if the user leaves the field.",
 		shortDesc: "Prevents the target from switching out.",
 		id: "spiritshackle",
+		isViable: true,
```

Comment:
Explanation: 80BP Ghost type move (and Ghost lacks good Ghost type moves). The effect seems quite powerful too.